app/assets/stylesheets/trips.css.scss

```diff
@@ -1,3 +1,12 @@
+#map{
```

Comment:
Avoid using id selectors<br>Opening curly brace `{` should be preceded by one space